vlm/src/main/scala/RasterSource.scala

```diff
@@ -82,4 +82,8 @@ trait RasterSource extends Serializable {
     def readBounds(bounds: Traversable[GridBounds]): Iterator[Raster[MultibandTile]] =
         bounds.toIterator.flatMap(read(_, (0 until bandCount)).toIterator)
 
+    def readPaddedTiles(tiles: Traversable[PaddedTile], bands: Seq[Int]): Iterator[Raster[MultibandTile]]
```

Comment:
We shouldn't add to the `RasterSource` interface like this. In the current formulation `PaddedTile` contains information about the pixel space from the `RasterSource` and some target layout. In general `RasterSource` should not be expected to handle data that is not directly related to the raster its sourcing.